README.adoc

```diff
@@ -136,39 +143,39 @@ In the [hotspot=17-22 file=2]`SystemClient` interface, add a response exception
 
 Error handling is an important step to ensure that the application can fail safely. If there is an error response such as `404 NOT FOUND` when invoking the remote service, you need to handle it. First, define an exception, and map the exception with the error response code. Then, register the exception mapper in the client interface.
```

Comment:
change
you need to handle it. First, define an exception, and map the exception with the error response code. Then, register the exception mapper in the client interface.
TO
**you need to define an exception and map the it with the error response code. Then, register the exception mapper in the client interface.**

Reply:
The original is clear.